jitpack.yml

```diff
@@ -1,2 +1,5 @@
-jdk:
-  - openjdk11
\ No newline at end of file
+before_install:
+  - source "$HOME/.sdkman/bin/sdkman-init.sh"
+  - sdk update
+  - sdk install java 17.0.7-tem
+  - sdk use java 17.0.7-tem
```

Comment:
This looks like something custom for your machine (sdk is not installed by default). Revert and use `- openjdk17`